src/services/GenericService.js

```diff
@@ -0,0 +1,28 @@
+
+class GenericService {
+  constructor(model) {
+    this._model = model
+  }
+
+  get = id => {
+    return this._model.findById(id)
```

Comment:
remember that you can only do: 
```
get = this._model.findById
``` 😄

Reply:
> remember that you can only do:

I think you wanted to say `... you can just 'simply' do:`.  
Yep I think I'm still not used to use JS as a FPL